src/components/Footer.js

```diff
@@ -14,7 +14,7 @@ const Footer = () => {
                 <div className="px-8 mx-auto max-w-page">
                     <div className="flex flex-col-reverse pb-5 md:flex-row items-center justify-between h-20 px-4 text-sm font-bold text-gray-800 dark:text-white font-rubik">
                             <CustomLink className="flex-shrink-0 text-center" href="https://github.com/skyline-emu/website/blob/master/LICENSE.md">
-                                © 2022 Skyline Team and Contributors
+                                © 2023 Skyline Team and Contributors
```

Comment:
Why to update it manually every year? We can use `{new Date().getFullYear()}`

Reply:
```
const YEAR = new Date().getFullYear()

            © <time>{YEAR}</time> Skyline Team and Contributors
```